cmd/lotus-miner/sealing.go

```diff
@@ -91,7 +91,10 @@ var sealingWorkersCmd = &cli.Command{
 
 			var barCols = uint64(64)
 			cpuBars := int(stat.CpuUse * barCols / stat.Info.Resources.CPUs)
-			cpuBar := strings.Repeat("|", cpuBars) + strings.Repeat(" ", int(barCols)-cpuBars)
+			cpuBar := strings.Repeat("|", cpuBars)
+			if int(barCols)-cpuBars >= 0 {
+				cpuBar = cpuBar + strings.Repeat(" ", int(barCols)-cpuBars)
```

Comment:
nit `cpuBars += ...`

Reply:
we don't know that in many places in lotus so i assumed it was bad go-ing

Reply:
but happy to agree

Reply:
```suggestion
				cpuBar += strings.Repeat(" ", int(barCols)-cpuBars)
```